broker-cli/commands/order/summary.js

```diff
@@ -91,6 +93,11 @@ async function summary (args, opts, logger) {
     // We extend the gRPC deadline of this call because there's a possibility to return
     // a lot of records from the endpoint.
     const orders = await brokerDaemonClient.orderService.getBlockOrders(request, { deadline: grpcDeadline(ORDER_SUMMARY_RPC_DEADLINE) })
+
+    if (json) {
+      return logger.info(JSON.stringify(orders.blockOrders, null, 2))
```

Comment:
[nit]
In `createUI`, we use `console.log` instead of logger (and elsewhere in the CLI). Do we want to stay consistent?

This shouldn't have any affect on output.

Reply:
Probably not in scope for this PR

Reply:
@bradleystachurski I lied. It is a relatively small change. What i've done is modify `createUI` to return a string... and then we log with `logger.info`